site/content/docs/5.2/forms/input-group.md

```diff
@@ -245,6 +275,12 @@ Input groups include support for custom selects and custom file inputs. Browser
 
 ### Custom select
 
+{{< ods-incompatibility-alert >}}
+These **3 first** form variants should not be used because they do not respect the Orange Design System specifications.
+
+Please refer to the [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) guidelines and to the [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) form examples on the Orange Design System website.
```

Comment:
Can't we redirect on Boosted select ?

Reply:
Why did you set this question as resolved? Is the question still valid or did you decided to ignore it at the end? It's just to know if I modify the callout message by adding a link towards Boosted Select page.

Reply:
The issue here is that we don't explain why these variants don't respect ODS specifications. Is it a problem of layout? Why the third one works and not the fourth one. Honestly, even by knowing the project pretty well, I don't understand why for the latter.